app.yaml.base

```diff
@@ -38,15 +38,10 @@ handlers:
           static_files: static/\1
           upload: static/(.*\.(bmp|gif|ico|jpeg|jpg|png))
 
-# site root
-        - url: /
-          static_files: static/index.html
-          upload: static/index.html
-          expiration: "15m"
-
 # All URLs for Back End logic
-        - url: /api/.*
+        - url: /.*
```

Comment:
> Do you have any suggestion in this case?

Stick to handling `/api/.*` here and tweak the static routes below to handle them via backend code.

Also while you're at it, handle 404's correctly.